### ISSUE TITLE
Support for git revert

### ISSUE THREAD

**Issue by maks** [maks/MGit#27]
_From @henrik242 on February 29, 2016 16:16_

The only feature I really miss is git revert.   No git client for Android has this :(

_Copied from original issue: sheimi/SGit#173_

**Comment by maks** [comment]
There is already  jgit top-level API support for Revert: https://github.com/eclipse/jgit/blob/master/org.eclipse.jgit.test/tst/org/eclipse/jgit/api/RevertCommandTest.java

so this should be fairly straight forward as a new Task per existing repo.tasks

**Comment by maks** [comment]
_From @henrik242 on May 26, 2016 5:38_

Great! Cool to see that sgit is maintained again.

**Comment by maks** [comment]
@henrik242 thanks for the suggestion. I've taken over as the maintainer for SGit, I did a new release a couple of days ago - I'm aiming to get 1.3.1 out in the next week or so and then releases on a roughly monthly cycle.

**Comment by maks** [comment]
_From @henrik242 on September 14, 2016 13:31_

@maks Any news? :)

**Comment by maks** [comment]
@henrik242 sorry about the delay - I've not had as much time as I expected over last few weeks so the 1.3.3 release has been delayed and so 1.3.4 is pushed out further as well. I'll update the 1.3.4 milestone due date to reflect this.

**Comment by henrik242** [comment]
Somehow I was under the impresson that #26 had to be solved for this issue to be implemented.  But no.  So that's good news :)  https://github.com/eclipse/jgit/blob/stable-3.7/org.eclipse.jgit/src/org/eclipse/jgit/api/RevertCommand.java

**Comment by maks** [comment]
@henrik242 yes thats correct, updating to latest jgit isn't a prerequiste for this. I'm prioritising this for the next release.

**Comment by henrik242** [comment]
@maks: Any progress on this?

**Comment by maks** [comment]
@henrik242 sorry not as yet, I've been caught up with work and other projects so have not had a chance to get to this yet.

**Comment by henrik242** [comment]
@maks How about now? :)

**Comment by henrik242** [comment]
Any news? :) @maks

**Comment by maks** [comment]
sorry @henrik242 I've still not had much time to spend on feature dev for MGit lately.

Also what time I have is not spent as productively as it could be due to the the rather aged architecture MGit inherited. I've tried updating gradually to nicer/newer architecture and stack but thats actually more work in the short term, so I'm starting work on a v2 that will be a from scratch rewrite using a modern arch and tech stack. All new feature development by me will there going forward but I'm still happy to accept contributions for the current v1 code base.
